### ISSUE TITLE
Seem like the gas amount too less or just missing zeroes?

### ISSUE THREAD

**Issue by alan-choo** [chewonithard/claimArbTokens#4]
Hi, for the gas to be sent to the compromised wallet (stated as 2000000000000 in line 55), is it missing two zeroes if refer to your comment stated at the side? Is it the correct value should be 200000000000000 ? If I am not wrong, ethers on Arbitrum network also have 18 decimals.

So I think 2000000000000 = 0.000002 eth only, which definitely too less for any transaction. Or maybe I'm missing something?

**Comment by alan-choo** [comment]
Another thing is, is it possible to add the function to monitor for block 16890400, and auto start the transaction when block number 16890400 reached inside the code?  I saw your instruction steps 4 stated go to BeaconCha to monitor the block, but would it be too slow for this way since execution speed is crucial in this case?

**Comment by monstertrakk** [comment]
> Hi, for the gas to be sent to the compromised wallet (stated as 2000000000000 in line 55), is it missing two zeroes if refer to your comment stated at the side? Is it the correct value should be 200000000000000 ? If I am not wrong, ethers on Arbitrum network also have 18 decimals.
> 
> So I think 2000000000000 = 0.000002 eth only, which definitely too less for any transaction. Or maybe I'm missing something?

Yes, that's right. That's a mistake, there are zeros missing.

**Comment by giraffe0x** [comment]
Yup I fixed the value. Anyway, it should be adjusted accordingly to what you think is necessary.

I can't find a solution for auto running the script at block 16780400, open to suggestions

**Comment by gjeanmart** [comment]
I think you should be able to do something like this
```
mainnetProvider.on("block", async (blockNumber) => {
  if(blockNumber === 16780400) {
     //do stuff
  }
})
```
